src/runtime_src/core/common/drv/include/kds_core.h

```diff
@@ -57,12 +57,27 @@ struct kds_ctx_info {
 /* TODO: PS kernel is very different with FPGA kernel.
  * Let's see if we can unify them later.
  */
+struct scu_usages {
+        u32                       succ_cnt;
+        u32                       err_cnt;
+        u32                       crsh_cnt;
+};
+
+struct sk_mem_stats {
+        u32                       hbo_cnt;
+        u32                       mapbo_cnt;
+        u32                       unmapbo_cnt;
+        u32                       freebo_cnt;
+};
+
 struct kds_scu_mgmt {
 	struct mutex		  lock;
 	int			  num_cus;
 	u32			  status[MAX_CUS];
 	u32			  usage[MAX_CUS];
 	char			  name[MAX_CUS][32];
+	struct scu_usages	  usages_stats[MAX_CUS];
```

Comment:
Could you Merge "usage" to scu_usages structure?

Reply:
Yes. We can merge.

Reply:
Done